sharding/nodesCoordinator/indexHashedNodesCoordinatorRegistry.go

```diff
@@ -212,6 +212,7 @@ func epochValidatorsToEpochNodesConfig(config EpochValidatorsHandler) (*epochNod
 		if err != nil {
 			return nil, err
 		}
+		result.lowWaitingList = configWithAuction.GetLowWaitingList()
```

Comment:
Can we rename these variables to reflect these are bool variables (e.g.: is/wasLowWaitingList?) otherwise, it looks like we are querying a list of nodes